tests/sentry/backup/test_releases.py

```diff
@@ -108,9 +108,9 @@ def test_at_23_10_0(self):
             with open(tmp_path, "rb") as f:
                 import_in_global_scope(f)
 
-    def test_at_23_9_1(self):
+    def test_at_23_10_0(self):
```

Comment:
should this be 23.10.1?

Reply:
It *should* have been, but I guess we didn't make this test at that release. Unless we go back and do so, I think this should mark the version it actually tests, not the latest one of that minor version chain. I'll try to be more thorough with point releases in the future.